spec/models/condition_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Condition, type: :model do
+
```

Comment:
suggested additions to the specification:

`describe '#name' do`
`  it 'can be empty' do...`  (or not)
`end
- also check `nil` condition.  what happens? is it valid?

and the same for all of the other instance vars.

Is most important for  `timing` and `config`.  Is an empty list considered valid?  (yes, I think, but it needs to explicitly be in the specification)

Reply:
I added some tests for this.